incubator/java-spring-boot2/image/Dockerfile-stack

```diff
@@ -8,22 +15,42 @@ RUN  apt-get -qq update \
   && apt-get -qq clean \
   && rm -rf /tmp/* /var/lib/apt/lists/*
 
-# Kotlin support
-COPY ./project/stack.properties /project/stack.properties
-RUN cd /usr/lib && \
-    KVER=$(cat /project/stack.properties | grep kotlin.version | sed 's/kotlin.version=//') && \
-    wget -q https://github.com/JetBrains/kotlin/releases/download/v$KVER/kotlin-compiler-$KVER.zip && \
-    unzip kotlin-compiler-*.zip && \
-    rm kotlin-compiler-*.zip && \
-    rm -f kotlinc/bin/*.bat  
+ENV APPSODY_STACK=${DOCKERHUB_ORG}/${STACK_ID}:${MAJOR_VERSION}.${MINOR_VERSION}
+LABEL appsody.stack="${APPSODY_STACK}"
 
 COPY ./project /project
+COPY ./templates /templates
```

Comment:
I'd prefer to not change template location..  in the first pass..

Reply:
We have to do this for the stack image to propagate the values down into the templates. The two go together.